common/src/chain/pos.rs

```diff
@@ -60,6 +60,7 @@ pub struct PoSChainConfig {
     difficulty_change_limit: PerThousand,
     /// Version of the consensus protocol
     consensus_version: PoSConsensusVersion,
+    token_issuance_version: TokenIssuanceVersion,
```

Comment:
Why is this here?

Reply:
To be able to check which version of tokens is allowed to use

Reply:
Because we already might have tokens in testnet, so I thought that's the way to allow v0 that already been issued but forbid it after netupgrade

Reply:
This should be another net upgrade in chain config that's tested elsewhere, not using consensus.

Reply:
aaah I see. Ok I'll add consensus agnostic net upgrade in chain config

Reply:
done